internal/proxy/proxy_config.go

```diff
@@ -75,6 +76,7 @@ type OptionsConfig struct {
 	HeaderOverrides map[string]string `yaml:"header_overrides"`
 	SkipAuthRegex   []string          `yaml:"skip_auth_regex"`
 	AllowedGroups   []string          `yaml:"allowed_groups"`
+	TLSVerify       *bool             `yaml:"tls_verify"`
```

Comment:
I think it still makes sense to use `TLSSkipVerify` as a `bool`, which removes the need for any default setting on load of the service configs.

Reply:
OK, I'll change it.